src/hooks/useLiveSpecsExt.ts

```diff
@@ -13,7 +13,7 @@ export interface LiveSpecsExtQuery {
     last_pub_id: string;
 }
 
-export interface LiveSpecsExt_MaterializeCapture {
+export interface LiveSpecsExt_MaterializeOrTransform {
```

Comment:
I wonder if we should start moving some of the naming conventions away from the entities to more descriptive adjectives. I just have no clue what adjective would make it clear you can `This live spec can be used in something else from the collections table`.

Also, this could easily end up bad if we make a naming convention that does not match other teams's vernacular.

Reply:
For the record, I am not a fan of the iterations of this interface name. What is here is the most logical adaptation of the existing name because I could not think of a better refactoring given the narrow scope.